src/app/app.component.ts

```diff
@@ -56,12 +56,22 @@ export class AppComponent implements OnInit, OnDestroy {
       this.apiKey = params.get('sdkKey');
       this.baseUrl = params.get('baseUrl');
       this.featureFlagKey = params.get('featureFlagKey');
+      let hideControls = params.get('hideControls');
 
-      if (!this.featureFlagKey) { this.featureFlagKey = 'isAwesomeFeatureEnabled'; }
+      if (!this.featureFlagKey) { this.featureFlagKey = ''; }
       this.apiKeyFormGroup = this.formBuilder.group({ apiKey: [this.apiKey, Validators.required] });
       this.featureFlagKeyFormGroup = this.formBuilder.group({ featureFlagKey: ['', Validators.required] });
```

Comment:
```suggestion
      this.featureFlagKeyFormGroup = this.formBuilder.group({ featureFlagKey: [this.featureFlagKey, Validators.required] });
```

Reply:
I have tried this, but it did not work as expected.
Maybe we can check it together in a call...